### PR TITLE
Add certified engineer feature

### PR DESCRIPTION
**Pull request by mvukovicbdit**
https://trello.com/c/RfALn6a8/46-bdit-certified-engineer

[Image component](https://nextjs.org/docs/pages/api-reference/components/image#responsive-image-with-aspect-ratio) is a bit weird, the only way I could resize the image while preserving its aspect ratio was to use both sizes and style props. If there is a better way, I'm open to suggestions. Don't hate me for doing frontend 😂

### CODE REVIEW

File: app/(dashboard)/edit/[id]/page.tsx [hunk 1]
```diff
@@ -13,6 +14,20 @@ const EditCvPage = async ({ params: { id } }: EditCvPageProps) => {
...
+  } = await supabase.auth.getSession();
+
+  const userId = session?.user.id;
+
+  const { data } = await supabase
+    .from("admins")
+    .select("id")
+    .filter("user_id", "eq", userId);
+
+  const isAdmin = data?.length && data.length > 0 ? true : false;
```
Comment (ridjis) [inline app/(dashboard)/edit/[id]/page.tsx]: I guess this is the same as

```suggestion
  const isAdmin = data?.length > 0;
```

File: app/(dashboard)/edit/[id]/page.tsx [hunk 2]
```diff
@@ -13,6 +14,20 @@ const EditCvPage = async ({ params: { id } }: EditCvPageProps) => {
...
+  const {
+    data: { session },
+  } = await supabase.auth.getSession();
+
+  const userId = session?.user.id;
+
+  const { data } = await supabase
+    .from("admins")
+    .select("id")
+    .filter("user_id", "eq", userId);
```
Comment (ridjis) [inline app/(dashboard)/edit/[id]/page.tsx]: [If I understood correctly](https://supabase.com/docs/reference/javascript/filter#:~:text=This%20is%20an%20escape%20hatch%20%2D%20you%0Ashould%20use%20the%20specific%20filter%20methods%20wherever%20possible.), this should be 
```suggestion
    .eq("user_id", userId);
```

**Review by ridjis — COMMENTED** [review]
Looks good overall - I've pushed commit with image updates, please take a look if it looks visually as what you expected.

Also, please take a look if we can simplify code further with those 2 suggestions I left.

File: app/(dashboard)/edit/[id]/page.tsx (hunk 1 above)
Comment (mvukovicbdit) [reply app/(dashboard)/edit/[id]/page.tsx]: not really cuz the suggested line can be undefined if the data is undefined resulting in a ts error

File: app/(dashboard)/edit/[id]/page.tsx (hunk 2 above)
Comment (mvukovicbdit) [reply app/(dashboard)/edit/[id]/page.tsx]: agree

File: app/(dashboard)/edit/[id]/page.tsx (hunk 1 above)
Comment (bokykisac) [reply app/(dashboard)/edit/[id]/page.tsx]: what about `const isAdmin = !!data?.length;` ?

Comment (ridjis) [reply app/(dashboard)/edit/[id]/page.tsx]: That would work, but I've pushed already change, used ternary operator as it's more readable
```typescript
const isAdmin = data ? data.length > 0 : false;
``` 

Thanks for suggestion tho @bokykisac.

**Pull request merged**